cdap-ui/app/features/experimental/templates/directives-playground.html

```diff
@@ -32,7 +60,7 @@
   </div>
 </div>
 
-<div class="panel panel-default">
+<!-- <div class="panel panel-default">
```

Comment:
I just commented it out because the field focus kept on making the scroll go to the focus